src/App.js

```diff
@@ -1,43 +1,28 @@
-import "./App.css";
-import logo from "./logo.png";
+import "./styles.css";
+import Navigation from "./Components/Navigation";
+import Home from "./pages/Home";
+import ProductPage from "./pages/ProductPage";
+import WishList from "./pages/WishList";
+import MyCart from "./pages/MyCart";
+import Mockman from "./pages/Mockman";
+import Login from "./pages/Login";
+import Signup from "./pages/Signup";
```

Comment:
CODE Suggestion:
Instead of adding routes in the App.js file, you could move it to a separate file say AppRoutes.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable easy to understand.

Reply:
Thanks Roshan. I will make route file separate in next PR.